python/arcticdb/adapters/rocksdb_library_adapter.py

```diff
@@ -0,0 +1,78 @@
+"""
+Copyright 2023 Man Group Operations Limited
+
+Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
+
+As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+"""
+import re
+import os
+
+# from dataclasses import dataclass
+
+
+from arcticdb.options import LibraryOptions
+from arcticc.pb2.storage_pb2 import EnvironmentConfigsMap, LibraryConfig
+from arcticdb.version_store.helper import add_rocksdb_library_to_env
+from arcticdb.config import _DEFAULT_ENV
+from arcticdb.version_store._store import NativeVersionStore
+from arcticdb.adapters.arctic_library_adapter import ArcticLibraryAdapter, set_library_options
+from arcticdb.encoding_version import EncodingVersion
+from arcticdb_ext.storage import CONFIG_LIBRARY_NAME
+
+
+class RocksDBLibraryAdapter(ArcticLibraryAdapter):
+    """
```

Comment:
Can you document in here that this is a work in progress, beta API and should not be relied upon?
And also log some warnings if anyone does actually use the RocksDB adapter so they see at runtime we're not making any promises about it.